Solutions/Corvus.Azure.Storage.Tenancy/Microsoft/Extensions/DependencyInjection/TenancyBlobStorageServiceCollectionExtensions.cs

```diff
@@ -30,48 +29,33 @@ public static class TenancyBlobStorageServiceCollectionExtensions
         }
 
         /// <summary>
-        /// Adds services required by tenancy Azure storage based stores, and configures the default
-        /// tenant's default storage account settings based on configuration settings.
+        /// Add components for constructing tenant-specific blob storage containers.
         /// </summary>
-        /// <param name="services">The service collection.</param>
+        /// <param name="services">The target service collection.</param>
         /// <param name="getOptions">Function to get the configuration options.</param>
-        /// <returns>The modified service collection.</returns>
+        /// <returns>The service collection.</returns>
         public static IServiceCollection AddTenantCloudBlobContainerFactory(
-            this IServiceCollection services,
-            Func<IServiceProvider, TenantCloudBlobContainerFactoryOptions> getOptions)
+            this IServiceCollection services, Func<IServiceProvider,
```

Comment:
You appaer to have split the type name `Func<IServierProvider, TenantCloudBlobContainerFactoryOptions>` across two lines! I think the newline should appear before the first `,` (i.e. before the `Func`).

Reply:
Weird.